# THE MEDIAN TEST

The median test is a non-parametric test used to compare the medians of two independent samples. It is a simple and easy-to-use test that can be used to detect differences in medians even when the data is not normally distributed.

#### 13.5.1 

Fifteen patient records from each of two hospitals were reviewed and assigned a score designed to
measure level of care. The scores were as follows:

Would you conclude, at the .05 level of significance, that the two population medians are different?
Determine the p value.


In [6]:
# library("coin") # is not necesari if i don use median_test

A <- c(99, 85, 73, 98, 83, 88, 99, 80, 74, 91, 80, 94, 94, 98, 80)
B <- c(78, 74, 69, 79, 57, 78, 79, 68, 59, 91, 89, 55, 60, 55, 79)

w <- sort(c(A, B))
mean_w = mean(w)
A_above <- length(A[A>mean_w])
A_below <- length(A[A<mean_w])
B_above <- length(B[B>mean_w])
B_below <- length(B[B<mean_w])


data <- matrix(c(A_above, A_below, B_above, B_below), nrow = 2, byrow = TRUE)
colnames(data) <- c("Above", "Below")
rownames(data) <- c("A", "B")

# Convert the matrix to a data frame
data <- as.data.frame(data)
data


# Perform the median test
# median_test(as.numeric(data$Above) ~ as.factor(rownames(data)))

chisq.test(data, correct = F)


Above Below
A 13     2   
B  2    13


	Pearson's Chi-squared test

data:  data
X-squared = 16.133, df = 1, p-value = 5.904e-05


At the 0.05 level of significance, there is sufficient evidence to reject the null hypothesis that the distribution of scores (above and below the mean) is the same for both groups A and B. The p-value of 5.904e-05 is smaller than the significance level of 0.05, indicating a statistically significant difference.

Therefore, we can conclude that the two population distributions represented by groups A and B have different proportions of scores above and below the mean.

#### 13.5.2

The following serum albumin values were obtained from 17 normal and 13 hospitalized subjects.

Would you conclude at the .05 level of significance that the medians of the two populations sampled
are different? Determine the p value.

In [8]:
A <- c(2.4,3.5,3.1,4.0,4.2,3.0,3.2,3.5,3.8,3.9,3.4,4.5,5.0,2.9,4.0,3.5,3.6)
B <- c(1.5,2.0,3.4,1.7,2.0,3.1,1.3,1.5,1.8,2.0,3.8,3.5,1.5)

w <- sort(c(A, B))
mean_w = mean(w)
A_above <- length(A[A>mean_w])
A_below <- length(A[A<mean_w])
B_above <- length(B[B>mean_w])
B_below <- length(B[B<mean_w])


data <- matrix(c(A_above, A_below, B_above, B_below), nrow = 2, byrow = TRUE)
colnames(data) <- c("Above", "Below")
rownames(data) <- c("A", "B")

# Convert the matrix to a data frame
data <- as.data.frame(data)
data

chisq.test(data, correct = F)

Above Below
A 14    3    
B  4    9


	Pearson's Chi-squared test

data:  data
X-squared = 8.1674, df = 1, p-value = 0.004265


At the 0.05 level of significance, there is sufficient evidence to reject the null hypothesis that the medians of the two populations sampled are the same. The p-value of 0.004265 is smaller than the significance level of 0.05, indicating a statistically significant difference.

Therefore, we can conclude that the medians of the two populations are different.